src/taipy/config/_config_comparator/_config_comparator.py

```diff
@@ -94,7 +94,7 @@ def __get_config_diff(self, config_1, config_2):
         json_config_1 = json.loads(_JsonSerializer._serialize(config_1))
         json_config_2 = json.loads(_JsonSerializer._serialize(config_2))
 
-        config_deepdiff = DeepDiff(json_config_1, json_config_2)
+        config_deepdiff = DeepDiff(json_config_1, json_config_2, ignore_order=True)
```

Comment:
Simpler than expected

Reply:
True. Deepdiff has already solved this problem for us 🤣